trade-finance-manager-ui/styles/layout.scss

```diff
@@ -56,4 +56,22 @@
       align-self: center;
       text-align: right;
     }
+}
+
+.ukef-with-separators {
+  > *:not(:last-child) {
+    border-bottom: 1px solid $govuk-border-colour;
+  }
+
+  @for $spacing-unit from 1 through 9 {
+    &.ukef-separator-spacing-#{$spacing-unit} {
+      > *:not(:first-child) {
+        @include govuk-responsive-padding($spacing-unit, "top");
+      }
+
+      > *:not(:last-child) {
+        @include govuk-responsive-padding($spacing-unit, "bottom");
+      }
+    }
+  }
```

Comment:
This allows us to do any of:

* ukef-separator-spacing-1
* ukef-separator-spacing-2
* ukef-separator-spacing-3
* ukef-separator-spacing-4
* ukef-separator-spacing-5
* ukef-separator-spacing-6
* ukef-separator-spacing-7
* ukef-separator-spacing-8
* ukef-separator-spacing-9

without having to create a separate rule for each